server/src/main/resources/application.yml

```diff
@@ -47,6 +47,9 @@ spring:
     # Don't check for templates on startup
     mustache:
         check-template-location: false
+    threads:
+        virtual:
+            enabled: false
```

Comment:
Set default to false to control on deployment and not to break previous behavior (in case of any problem)